### ISSUE TITLE
R-4.2.0 arcgisbinding not available

### ISSUE THREAD

**Issue by bangen** [R-ArcGIS/r-bridge-install#109]
I recently updated to R version 4.2.0 and am running ArcGIS Pro version 2.9.3.  

I have not been able to install the arcgisbinding package.  Any insights/help would be greatly appreciated.  Thanks!

Code:

```
install.packages("arcgisbinding", repos="https://r.esri.com", type="win.binary")
``` 

Returns message:
```
Warning: unable to access index for repository https://r.esri.com/bin/windows/contrib/4.2:
  cannot open URL 'https://r.esri.com/bin/windows/contrib/4.2/PACKAGES'
Warning message:
package ‘arcgisbinding’ is not available as a binary package for this version of R
```

**Comment by orhuna** [comment]
@bangen we are releasing v300 that will solve this issue. Version 245 will not support R 4.2. I will update here once it is out.

**Comment by orhuna** [comment]
@bangen `arcgisbinding` v300 is now released. Please get the latest version of the package and retry.

**Comment by bangen** [comment]
@orhuna Related question.  Are all arcgisbinding functions available in R 4.2 since it is a 64 bit version?

**Comment by orhuna** [comment]
@bangen yes. We have full support for 64-bit R with the base package being built against 64-bit. 

Since your original question, we released `arcgisbinding v300` that has full support for R 4.2. If you have not updated your `arcgisbinding`, or if `arc.check_product()` is not returning 300 for `arcgisbinding` version, I highly recommend updating.
